benchmarks/src/main/java/io/grpc/benchmarks/SocketAddressValidator.java

```diff
@@ -31,12 +31,14 @@
 
 package io.grpc.benchmarks;
 
+import com.google.errorprone.annotations.Immutable;
 import java.net.InetSocketAddress;
 import java.net.SocketAddress;
 
 /**
  * Verifies whether or not the given {@link SocketAddress} is valid.
  */
+@Immutable
 public interface SocketAddressValidator {
```

Comment:
The current code is fine and valid. Yes, we may want it to be a different way, but just adding the `@Immutable` is the least invasive and equivalent to swapping to an enum. I see rewriting code unnecessarily as counter-productive, because nobody will agree exactly how it should be implemented and so things can be rewritten again and again without bound.